baseplate/sidecars/event_publisher.py

```diff
@@ -116,7 +116,9 @@ def serialize(self) -> SerializedBatch:
 
 publishCountTotal = Counter("event_publish_total", "total count of published events")
 publishLatency = Histogram("event_publish_latency", "latency for publishing a batch of events")
-publishErrors = Counter("event_publish_errors", "total count of published events", ["error_type"])
+publishErrors = Counter(
+    "event_publish_errors", "total count of errors when publishing events", ["error_type"]
```

Comment:
please follow https://prometheus.io/docs/practices/naming/ for naming convention.

all metrics should be suffixed with the unit (whenever applicable), then counters should be suffixed with `_total` after the unit.

so the 3 here should be: `event_publishes_total` (or maybe `event_publish_attempts_total`?), `event_publish_latency_seconds`, and `_event_publish_errors_total` respectively. similar to other sidecars.